src/TConsole.h

```diff
@@ -254,7 +254,9 @@ class TConsole : public QWidget
     QString profile_name;
     TSplitter* splitter;
     bool mIsPromptLine;
-    QToolButton* logButton;
+    QToolButton* mTimeStampButton; // was an auto (local to constructor) pointer timeStampButton
```

Comment:
Don't need to know what it was before. `git blame` can help us with that.

Reply:
When I am coding and changing a member name around so that I can remember what it was called at the start of my ~~hacking~~coding session - but yeah I can see how it leads to historical cruft...

Reply:
:+1:

Reply:
☑️ Have cleaned out a number of `// Was` type entries I had left behind - but I ought to document variable scope/renames in the commit messages as I find it useful when trying to merge/rebase/cherry-pick to remember what the identifier was at a particular point in time... 😌